.circleci/config.yml

```diff
@@ -474,7 +474,7 @@ jobs:
             mkdir -p ~/.ssh
             ssh-keyscan github.com >> ~/.ssh/known_hosts
             git fetch --all
-            RELEASE_BRANCH=$(git branch -a --list *release/contracts/* | tail -1 | sed -e 's/remotes\/origin\///g')
+            RELEASE_BRANCH=release/contracts/1
```

Comment:
won't this only pass after we merge the pre-release branch? or did we already?

Reply:
thanks for the review. My PR was not up to date because I keep forgetting to push to my fork (instead of this repo), from which I'm now issuing PR's womp. It's up to date now. It needs to point to the base release, from before any audit fixes. I'm just waiting to see how this affects the build. THen will need feedback on the best way to reference this base commit (I have it as a branch for now)